tests/integration/installation_test.go

```diff
@@ -55,143 +47,31 @@ var _ = Describe("Installation Lifecycle", func() {
 				},
 			}
 			Expect(k8sClient.Create(ctx, inst)).Should(Succeed())
-			Expect(waitForPorter(ctx, inst, "waiting for the bundle to install")).Should(Succeed())
-			validateInstallationConditions(inst)
+			Expect(waitForPorter(ctx, inst, 1, "waiting for the bundle to install")).Should(Succeed())
+			validateResourceConditions(inst.Status.Conditions)
 
 			patchInstallation := func(inst *porterv1.Installation) {
 				controllers.PatchObjectWithRetry(ctx, logr.Discard(), k8sClient, k8sClient.Patch, inst, func() client.Object {
 					return &porterv1.Installation{}
 				})
+				// Wait for patch to apply, this can cause race conditions
```

Comment:
I see a comment here but no corresponding code below. Does this comment apply to something I'm just missing or did you intend to add more waiting here?

Reply:
Whoops this was before I figured out the race conditions in the waitForPorter, will remove